Examples/Scripts/Python/material_recording.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+import os
+import warnings
+
+import acts
+from acts.examples import (
+    FixedVertexGenerator,
+    ParametricParticleGenerator,
+    FixedMultiplicityGenerator,
+    EventGenerator,
+    RandomNumbers,
+)
+
+import acts.examples.dd4hep
+import acts.examples.geant4
+import acts.examples.geant4.dd4hep
+
+u = acts.UnitConstants
+
+_material_recording_executed = False
+
+
+def runMaterialRecording(g4geo, outputDir, tracksPerEvent=10000, s=None):
```

Comment:
I am a bit confused by the `tracksPerEvent=10000` it doesn't seem to do anything

Reply:
It should do something now in the latest version of the branch.